[AND-3178] Mopub will return playback errors from Cache Busting

anton-blashchuk opened the pull request:
updated callbacks and logs to reflect situation better
https://vungle.atlassian.net/browse/AND-3178

Commit:
[AND-3178] Mopub will return playback errors from Cache Busting
updated callbacks and logs to reflect situation better
https://vungle.atlassian.net/browse/AND-3313

alexander-gubov commented on Vungle/src/main/java/com/mopub/mobileads/VungleRouter.java:
```diff
@@ -254,13 +254,15 @@ void playAdForPlacement(String placementId, @Nullable String adMarkup, AdConfig
                 VungleRouterListener routerListener = sVungleRouterListeners.get(placementId);
                 if (routerListener != null) {
-                    routerListener.onUnableToPlayAd(placementId, "Invalid/Inactive Placement Id");
+                    routerListener.onUnableToPlayAd(placementId, "Ad for Placement was busted");
```
how could you know that it was busted?

anton-blashchuk replied:
to say for sure we have to add a lot of code. here it is more like an educated guess, as very recent bust is the most probable explanation why this situation happened

Commit:
[AND-3178] Mopub will return playback errors from Cache Busting
unified callbacks and logs; removed mention of cache busting in logs; removed dead code in VungleBanner that's related to MRECs that are now handled by code just before deleted one
https://vungle.atlassian.net/browse/AND-3178
